other/install.php

```diff
@@ -1906,7 +1894,7 @@ function template_install_below()
 		</div></div>
 		<div id="footer">
 			<ul>
-				<li class="copyright"><a href="http://www.simplemachines.org/" title="Simple Machines Forum" target="_blank" class="new_win">SMF &copy; 2017, Simple Machines</a></li>
+				<li class="copyright"><a href="https://www.simplemachines.org/" title="Simple Machines Forum" target="_blank" class="new_win">SMF &copy; 2017, Simple Machines</a></li>
```

Comment:
For consistency with the rest of the codebase. Alternatively, @jdarwood007 could go through everything and change all the links to use https. But we should use one or the other consistently.